.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will install Python dependencies and run tests with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
```

Comment:
Similarly, this job does more than just build, could we rename it to 'build-and-test' or even just 'test'?